client/menus/colors.lua

```diff
@@ -6,21 +6,23 @@ local originalWindowTint
 local originalLabelIndex = 1
 local originalInterior
 local originalLivery = {}
+local config = require('config.client')
+local sharedConfig = require('config.shared')

```

Comment:
You don't need the () around it, function calls work without it and we use it without in these cases

Reply:
i blame copilot 😆